.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, 2.7.4, 3.0.0, 3.0.1, 3.0.2]
+        ruby: [2.7, 2.7.4, 3.0.0, 3.0.1, 3.1.0, 3.1.1]
```

Comment:
may be to add `head` version? and do we really need to test both  3.0.0 and 3.0.1 ?